### ISSUE TITLE
Adding prompts

### ISSUE THREAD

**Issue by egormly** [v8hid/infinite-zoom-automatic1111-webui#29]
is super annoying. I am not sure if it's gradio, my system or what but if I try to copy/paste, type or whatever the prompt box gets cleared and I have to click several times.  it just seems janky.

Can you just make it a standard text box?

**Comment by GeorgLegato** [comment]
your are right,
whatever textbox we use, we have just now added an export and import from file. 
go as: create 2 rows, Export Prompts, check the syntax and add your stuff i that json file.
hint: ask chatgpt to create an JSON-Array like [1,"text"], [2,"othertxt"] ...

**Comment by deklesen** [comment]
I also really don't like that textbox.  I frequently delete the content by accident trying to edit it, it doesnt support undo/redo, i don't see a reason why it should be sortable, and rows can't be deleted.

**Comment by GeorgLegato** [comment]
@deklesen would you like to update to recent version and test the Export/Import stuff? If you would accept a simple textarea, maybe that export/import is productive enough, before changing too much in this solution yet

**Comment by deklesen** [comment]
I am on the latest version, the import/export works fine..

I'd prefer a textbox where i enter the json to the current solution, yes...

But then we might have more ui issues down the road, i.e. it has been mentioned somewhere else that the json might contain all generation information (steps, seed once added, etc) and then when entering the json in a textarea its not obvious what should happen, i.e. should the sliders be updated? But then what if the sliders get changed, does the json content override it again when pressing generate?

I'd much prefer if we could have an input somewhat like the current (rows, but with better functionality) and allow import of the json, where upon import, the sliders and prompt table get loaded once, and then thats it.

Sorry for the wall of text 🤪

**Comment by v8hid** [comment]
@GeorgLegato @deklesen @egormly 
Don't worry guys, I'm working on a new custom smooth UI to make a lot of people happy :D

**Comment by GeorgLegato** [comment]
> @GeorgLegato @deklesen @egormly Don't worry guys, I'm working on a new custom smooth UI to make a lot of people happy :D

please try to reuse the imptr/export approach, was a lot work. we can simplify json without headers etc

**Comment by v8hid** [comment]
@GeorgLegato Don't worry, I just wanted to let you know that I appreciate all the effort you put into [project/task]. I reviewed it and I know how much work went into it. Thank you

**Comment by deklesen** [comment]
> @GeorgLegato @deklesen @egormly Don't worry guys, I'm working on a new custom smooth UI to make a lot of people happy :D

You're a saint! ❤️ 💪

**Comment by jlitz** [comment]
i am new  but same issue  prompt disappears when i attempt to  revise it

**Comment by GeorgLegato** [comment]
yes, that the nature of Gradio-Table. first we have to fix other technical stuff, then coming back for ui-improves. grad table is not really the best  one
